### DISCUSSION TITLE
Docsy and Lighthouse

### DISCUSSION THREAD

**Discussion by pa-rj** [google/docsy#2084] [Q&A]
Has anyone went thru optimizing docsy for lightouse rating?

I'm in particular interested in purgecss integration to remove unused css.

**Comment by fekete-robert** [comment]
Not recently, but in general you can get pretty decent ratings. Some tips:
- Certain issues have nothing to do with docsy, but depend on how you're hosting your site, for example, caching times, gzip compression. How or if you can change these depends on your hosting provider, for example, last time I checked you couldn't change caching times on github pages
- Make sure to sure the production environment of hugo to generate the site that you deploy. AFAIK Hugo performs certain optimizations (like css minification) only for this environment. And it must be called "production" (prod or similar names don't trigger the optimization)
- It's worth to check a few different pages and fix some of the main issues, unused css might or might not be significant
